raiden/tests/utils/factories.py

```diff
@@ -351,9 +351,9 @@ def make_hop_to_channel(channel_state: NettingChannelState = EMPTY) -> HopState:
 # In this module constants are in the bottom because we need some of the
 # factories.
 # Prefixing with UNIT_ to differ from the default globals.
-UNIT_SETTLE_TIMEOUT = 50
-UNIT_REVEAL_TIMEOUT = 5
-UNIT_TRANSFER_AMOUNT = 50
+UNIT_SETTLE_TIMEOUT = BlockExpiration(50)
+UNIT_REVEAL_TIMEOUT = BlockExpiration(5)
```

Comment:
Those should be `BlockTimeout`, right?

Reply:
Yes, I got confused by a test that used this as an expiration value.